src/app/pages/plugins/available-plugins/available-plugins.component.ts

```diff
@@ -16,7 +16,9 @@ export class AvailablePluginsComponent implements OnInit {
     @Input() parent: any;
 
     protected queryCall = 'plugin.available';
-    protected queryCallOption = {'plugin_repository': 'https://github.com/freenas/iocage-ix-plugins.git'};
+    protected queryCallOption = {
```

Comment:
I don’t follow. Middleware already returns the correct URI for the ixsystems collection in both FN and TN, why do we have to check this again and hard code the URI ?

Reply:
Thanks @sonicaj ! fixed it :)